bundles/io/org.eclipse.smarthome.io.transport.serial.javacomm/src/main/java/org/eclipse/smarthome/io/transport/serial/internal/JavaCommPortCreator.java

```diff
@@ -22,18 +23,47 @@
 import javax.comm.CommPortIdentifier;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.io.transport.serial.ProtocolType;
+import org.eclipse.smarthome.io.transport.serial.ProtocolType.PathType;
 import org.eclipse.smarthome.io.transport.serial.SerialPortIdentifier;
-import org.eclipse.smarthome.io.transport.serial.SerialPortManager;
+import org.eclipse.smarthome.io.transport.serial.SerialPortProvider;
 import org.osgi.service.component.annotations.Component;
 
 /**
- * Specific serial port manager implementation.
  *
- * @author Markus Rathgeb - Initial contribution
+ * @author MatthiasS
```

Comment:
Full name here as well please.
And as you seem to have copied/renamed this class, you might actually want to keep Markus as the original author.